src/_css/main.css

```diff
@@ -293,6 +293,14 @@ main a:hover, main a:focus, footer a:hover, footer a:focus {
 main a:hover code, main a:focus code {
   background-color: inherit;
 }
+.note a {
+  color: var(--darkblue);
+  border-bottom: 1px solid var(--darkblue);
```

Comment:
@mathiasbynens By the way, any reason you use `border` instead of [`text-decoration`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration) and [`text-decoration-skip-ink`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-skip-ink)?

Reply:
No

Reply:
Either way, let’s discuss that potential change as a separate patch, as it involves rules all over the stylesheet.

Reply:
@RReverser Switching bottom borders for proper text decoration might be a good follow-up PR.

Reply:
https://github.com/v8/v8.dev/issues/279